### PR TITLE
 EAHW-2813 - repository health-check

### PR DESCRIPTION
**Pull request by sewerynmi**

### CODE REVIEW

**Review by IanFisherHO — DISMISSED** [review]

**Review by JWeston-HO — DISMISSED** [review]

**Review by JWeston-HO — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,27 +1,50 @@
...
-$ mvn clean install
-```
+## Debugging
+
+Assuming you know how to debug java service, the principal is the same when using [callisto-localdev solution](https://github.com/UKHomeOffice/callisto-localdev). When running using docker compose, the debugger is exposed on the default port (50211). In the case of the debugger port conflicts, change both the ports and the entrypoint section in the docker-compose.yml
+
+## Kafka
 
-## Actuator Kafka Error Endpoint
-We have a counter configured for actuator which will give a metric on how many times deserialization has failed for kafka event messages.
```
Comment (JWeston-HO) [inline README.md:22]: This can still be here the actuator endpoint is still in the project

File: README.md [hunk 2]
```diff
@@ -1,27 +1,50 @@
...
+## Debugging
+
+Assuming you know how to debug java service, the principal is the same when using [callisto-localdev solution](https://github.com/UKHomeOffice/callisto-localdev). When running using docker compose, the debugger is exposed on the default port (50211). In the case of the debugger port conflicts, change both the ports and the entrypoint section in the docker-compose.yml
+
+## Kafka
 
-## Actuator Kafka Error Endpoint
-We have a counter configured for actuator which will give a metric on how many times deserialization has failed for kafka event messages.  
-This can be accessed at
+Project requires a Kafka instance with all required permissions to write to topic. If [running the service within LocalDev solution](#running-as-part-of-the-localdev-environment), no futher configuration is required.
```
Comment (JWeston-HO) [inline README.md:48]: This service doesn't write to the topic it reads or consumes

File: README.md (hunk 1 above)
Comment (sewerynmi) [reply README.md:22]: Added back

File: README.md (hunk 2 above)
Comment (sewerynmi) [reply README.md]: Fixed

**Review by JWeston-HO — APPROVED** [review]

**Review by Xeena-Newman — APPROVED** [review]

**Pull request merged**